Source/Scene/Label.js

```diff
@@ -825,38 +957,58 @@ define([
      * @private
      */
     Label.getScreenSpaceBoundingBox = function(label, screenSpacePosition, result) {
```

Comment:
I think so.  I worked on improving the code coverage for labels, the new code and some previously untested conditionals within older code are covered now.